### ISSUE TITLE
allow for secondary bookmarks 

### ISSUE THREAD

**Issue by makapuf** [cbeyls/fosdem-companion-android#30]
Sometimes I'd like to have lower priority bookmarks by example to keep a bookmark for later viewing or as a low priority alternative to an interesting talk. It would not show as a conflict by example.

**Comment by hrw** [comment]
Maybe not second bookmarks but priorities.

So I mark "docker python" as primary and "rake for dummies" and "go for mcu" as secondary. 

This way I know that "docker python" is one I want to go but it is raining so I go for "go for mcu" as it is dame building as I am.

**Comment by testman42** [comment]
Yes, some way to indicate the level of interest for a certain talk.
Would be useful for handling schedule conflicts.

**Comment by Ian2020** [comment]
Perhaps by default you get the current bookmark icon but optionally you can change the colour and/or shape if you tap again or tap and hold. This way a user can attach their own meaning to the symbols. For example you might use a red bookmark to mean high priority, amber for medium and green to indicate you have seen the talk etc etc. Colours alone might not be enough and be an accessibility issue for colour-blind users so being able to change the symbol as well like offering a star, cross, tick icon etc.
